lib/accredited_representation/seed_data.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentation
+  module SeedData
+    def self.create_rep_data(email, ogc_number, poa_code, individual_type)
+      accredited_individual = AccreditedIndividual.find_or_initialize_by(registration_number: ogc_number)
+      accredited_individual.update!(ogc_id: SecureRandom.uuid,
+                                    poa_code:,
+                                    individual_type:,
+                                    email:)
+
+      verified_rep = AccreditedRepresentativePortal::VerifiedRepresentative.find_or_initialize_by(email:)
```

Comment:
We'll need to rename `VerifiedRepresentative` here, given the rename we had in flight during this sprint! Ref:
https://github.com/department-of-veterans-affairs/vets-api/pull/17516/files